src/Microsoft.AspNetCore.Mvc.Razor/TagHelperComponent/ITagHelperComponent.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.TagHelperComponent
+{
+    public interface ITagHelperComponent
+    {
+        /// <summary>
+        /// Set to <c>true</c> if <see cref="ITagHelperComponent"/> applies to the given <see cref="TagHelperContext"/>.
```

Comment:
`Set` is typically used in reference to properties. Use `Returns <c>true> to indicate...`